learning_resources_search/serializers_test.py

```diff
@@ -88,9 +86,7 @@ def test_serialize_bulk_courses_for_deletion():
         serializers.serialize_bulk_courses_for_deletion([course.learning_resource_id])
     ) == [
         {
-            "_id": api.gen_course_id(
-                course.learning_resource.platform, course.learning_resource.readable_id
-            ),
+            "_id": course.learning_resource.id,
```

Comment:
One minor concern with using the database id is that if we mass delete resources then reimport them, the ids will be different, which could potentially mess up any saved query urls that include them, as opposed to a combo of platform and readable_id which should always be the same.  But that is true of querying id via REST API too anyway.